### PR TITLE
Stabilized toString output for MoneyContext and MoneyNumeric

### PR DESCRIPTION
**Pull request by garyKeorkunian**
* Added custom `toString` methods to `MoneyContext` and `MoneyNumeric` to stabilize their output.  This should prevent thrash in the README file on each run of `tut`.

No tests should be required for these methods, as they are primarily used in the REPL and production application code should generally not depend on these values.

### CODE REVIEW

**Comment by cquiroz** [comment]
This looks good, though I'm not sure about the assumption that it wouldn't impact application code. Often code relies on `toString` for better or worse. Perhaps we should add a note to the Readme/Release notes?

**Comment by garyKeorkunian** [comment]
Yes, I suppose some apps could be using in logs or other ways I've not anticipated.  Once it's merged, I'll add a migration note to the release history.

File: shared/src/main/scala/squants/market/MoneyContext.scala [hunk 1]
```diff
@@ -34,6 +35,26 @@ case class MoneyContext(
     allowIndirectConversions: Boolean = true) {
 
   /**
+    * Custom implementation using SortedSets to ensure consistent output
+    * @return String representation of this instance
+    */
+  override def toString: String = string
```
Comment (derekmorr) [inline shared/src/main/scala/squants/market/MoneyContext.scala]: I wonder if we could just map over `currencies` to get their symbols and make a `SortedSet` from that. And do the same for `rates`. That would avoid needing to create `Ordering` instances for `Currency` and `CurrencyExchangeRate`.

Something like:

```scala
val cSet = currencies.map { _.code }.sorted.mkString(',')
```

Comment (garyKeorkunian) [inline shared/src/main/scala/squants/market/MoneyContext.scala]: Yes, much cleaner.  I needed to through a toList call in the pipeline to provide sortability without the Ordering implicit.  I'll have a new push soon.

Comment (garyKeorkunian) [inline shared/src/main/scala/squants/market/MoneyContext.scala]: The SortedSet was overkill for the internal implementation, for sure.

**Pull request merged**